app/src/main/java/org/mozilla/focus/home/HomeTilesManager.kt

```diff
@@ -171,11 +171,11 @@ class CustomTilesManager private constructor(context: Context) {
     }
 
     @UiThread
-    fun unpinSite(context: Context, url: String): Boolean {
-        val tile = customTilesCache.remove(url) ?: return false
+    fun unpinSite(context: Context, url: String): String? {
```

Comment:
Add a javadoc style comment here that says when you return a null, same as above.